ivy/functional/backends/torch/linear_algebra.py

```diff
@@ -218,6 +218,16 @@ def matrix_power(
 matrix_power.support_native_out = True
 
 
+@with_unsupported_dtypes({"1.11.0 and below": ("float16", "bfloat16")}, backend_version)
```

Comment:
If a function isn't defined in the backend, we should define it as an Ivy API, and implement it for all backends.